src/Jupyter/Magic/Resolution/MagicResolver.cs

```diff
@@ -38,7 +39,10 @@ public MagicSymbolResolver(IServiceProvider services, ILogger<MagicSymbolResolve
             this.cache = new Dictionary<AssemblyInfo, MagicSymbol[]>();
             this.logger = logger;
 
-            this.kernelAssembly = new AssemblyInfo(typeof(MagicSymbolResolver).Assembly);
+            this.kernelAssemblies = new[] {
```

Comment:
Very minor, but other `new []` in this project have used brace-on-next-line. As with C# style comments in general, please feel free to disregard.
```suggestion
            this.kernelAssemblies = new[]
            {
```